src/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/EmptyWeb-CSharp/Program.cs

```diff
@@ -14,12 +14,11 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            BuildWebHost(args).Run();
+            CreateWebHostBuilder(args).Build().Run();
         }
 
-        public static IWebHost BuildWebHost(string[] args) =>
```

Comment:
Won't this break EF tooling? /cc @bricelam

Reply:
I believe EF tooling was updated by @javiercn .

Reply:
https://github.com/aspnet/EntityFrameworkCore/pull/10100

Reply:
Okay, so EF tooling now supports both patterns? So, this change will work with EF Core 2.1, but not with 2.0, but an app created using the 2.0 pattern will continue to work with 2.1? If so, I think that's okay.

Reply:
Yep